<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndiceCursos</class>
 <widget class="QDialog" name="IndiceCursos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>INDICE DE REGISTRO DE CURSOS</string>
  </property>
  <widget class="QLineEdit" name="txtConsultar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNuevo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Nuevo Curso</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConsultar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Consultar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="Lista">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>501</width>
     <height>241</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NOMBRE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>COSTO</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DETALLE</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnEditar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar Curso</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEliminar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar Curso</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
